app/src/main/java/grinnell/appdev/edu/lyles/ItemAdapter.java

```diff
@@ -0,0 +1,160 @@
+package grinnell.appdev.edu.lyles;
+
+import android.animation.ObjectAnimator;
+import android.content.Context;
+import android.support.v4.content.ContextCompat;
+import android.support.v4.widget.TextViewCompat;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageButton;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+
+import java.util.ArrayList;
+
+import grinnell.appdev.edu.lyles.preferences.FavoritesManager;
+
+/**
+ * An adapter for translating between an ArrayAdapter for MenuItems to a ListView
+ *
+ * @author Shelby Frazier
+ */
+
+public class ItemAdapter extends RecyclerView.Adapter<ItemAdapter.ViewHolder> {
+
+    public class ViewHolder extends RecyclerView.ViewHolder implements View.OnClickListener{
+        public TextView titleTextView;
+        public TextView priceTextView;
+        public ImageView imageView;
+        public ImageButton favoriteButton;
+        public TextView detailsTextView;
+
+        public ViewHolder(View itemView) {
+            super(itemView);
+            titleTextView = (TextView) itemView.findViewById(R.id.tvTitle);
+            priceTextView = (TextView) itemView.findViewById(R.id.tvPrice);
+            imageView = (ImageView) itemView.findViewById(R.id.ivFood);
+            favoriteButton = (ImageButton) itemView.findViewById(R.id.btnFavorite);
+            detailsTextView = (TextView) itemView.findViewById(R.id.tvDetails);
+
+            itemView.setOnClickListener(this);
+        }
+
+        @Override
+        public void onClick(View view) {
+            if (expandedIndex != -1) {
```

Comment:
What is -1 here? You might want to extract this out as a constant and give it a proper variable name.